core/src/main/scala/cromwell/core/path/EvenBetterPathMethods.scala

```diff
@@ -38,7 +38,7 @@ trait EvenBetterPathMethods {
 
   // betterFile.symbolicLink calls Files.readSymbolicLink, but then implicitly converts the java.nio.Path returned to a better.File
   // which calls toAbsolutePath. Consequently, if the path was relative, the current directory is used to make it absolute.
-  // This is not the desired behaviour to be able to follow relative symbolic links, so bypass better files method and directly use the java one.
+  // This is not the desired behavior to be able to follow relative symbolic links, so bypass better files method and directly use the java one.
```

Comment:
Seeing this makes me think that it should be fixed in the latest better files version. We could switch back to using `betterFile.symbolicLink`